### ISSUE TITLE
Spyder Vim plugin maintenance and Spyder Vim 0.1.0 release

### ISSUE THREAD

**Issue by dalthviz** [spyder-ide/spyder-vim#78]
Hi @ok97465, as commented at https://github.com/spyder-ide/spyder-vim/pull/77#pullrequestreview-1089042130 and as we started discussing at https://github.com/spyder-ide/spyder/issues/17915#issuecomment-1133776306 would you like to help with the plugin maintenance?

Besides that (and the issues already in the millestone), some pending things that I think we need to do before doing an initial release - `v0.1.0`:

* [x] Update `README.md` with information and a screenshot/gif to show how the plugin works. Update installation instructions and README content in general.
* [x] Add `RELEASE.md` as a guide for further releases (following the content of other working plugins repos)
* [x] Add `CONTRIBUTING.md` as a guide to contribute to the repo (following the content of other working plugins repos)

Also pinging @spyder-ide/core-developers in case someone else wants to help here, has any comment or suggestion :)

**Comment by ccordoba12** [comment]
I'm ok with this plan.

**Comment by ok97465** [comment]
If there's something I can help to maintain this repo, I'll do it.
:smiley:

**Comment by CAM-Gerlach** [comment]
Great, thanks @ok97465 ! Looks like @dalthviz listed some in the OP here, as well as those in the milestone :)

**Comment by ccordoba12** [comment]
I think @dalthviz can help with items 2 and 3 (i.e. adding `RELEASE.md` and `CONTRIBUTING.md`) and @ok97465 with the first one (i.e. updating `README.md` with an up to date screenshot), if you're ok with that, of course.

**Comment by dalthviz** [comment]
Note: Probably alongside getting this issue closed, before starting to do the actual release steps, some other issues should be closed too or maybe marked as superseded by this issue like #21 #22 #32 and #33

**Comment by ccordoba12** [comment]
I think #32 is duplicate of #22 and #33 of #21. If you want, you can close #21 as part of the 0.1.0 release and #22 after creating a conda-forge package, which would be out of the release.

**Comment by dalthviz** [comment]
So I think we are ready to do an initial release, would you like to check/do that process @ok97465 ?

The release process that needs to be done is described at https://github.com/spyder-ide/spyder-vim/blob/master/RELEASE.md (you will need to have a PyPI account to submit the distribution to PyPI and also close this issue and #21 before starting). Let us know if you want to do it!

Also, just in case, is there something else that could be missing @ccordoba12 @CAM-Gerlach ?

**Comment by ccordoba12** [comment]
I read the release file and it looks good to me.

**Comment by CAM-Gerlach** [comment]
Looks fairly good to me as well; at least content-wise it looks basically up to date with our modern version for QtPy, Docrepr, etc.

**Comment by dalthviz** [comment]
@ok97465 thank you for doing the release! Since v0.1.0 was already released I think we can now close this one, right? Also, @ok97465 could you add @ccordoba12 @CAM-Gerlach and me as maintainers on the PyPI package project please? Thanks again!

**Comment by ccordoba12** [comment]
> could you add @ccordoba12 @CAM-Gerlach and me as maintainers on the PyPI package project please?

Agreed. My PyPI username is ccordoba12.

**Comment by ccordoba12** [comment]
I'm going to close this issue to update our Changelog with it and #21.

**Comment by ok97465** [comment]
@CAM-Gerlach  Could you tell me the your username of PYPI?

**Comment by dalthviz** [comment]
Just in case, I think @CAM-Gerlach PyPI profile is this one: https://pypi.org/user/CAMGerlach/

**Comment by CAM-Gerlach** [comment]
Yup, that would be me :+1:
